### PR TITLE
Rip main controller

### PR DESCRIPTION
**Pull request by egedebruin**
The main controller is removed and its responsibilities are in other controllers.

The biggest problem is that somewhere all controllers need to be made. When you make the controllers in Main, you need to pass them via MonitorScene to VideoPane to the different panes. This creates unneeded links from the Controllers to the Scene and main.

Now I create the controllers in VideoPane, the problem now however is that VideoPane creates almost everything (so all the controllers and all the panes). Maybe you have ideas about this.

For the rest, look good if it works on everyting and if you agree with the refactoring.

### CODE REVIEW

**Comment by egedebruin** [comment]
tests failing :(

**Comment by lisv** [comment]
Kun je ook die vage filmpjes deleten (postit.mov en het nieuwsfilmpje)? En inputscene?

File: src/main/java/camera/CameraActivity.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public double addActivity(final Mat frame, final int partNumber,
             change += v;
         }
 
-        final int minFrames = 30;
+        final int minFrames = 50;
```
Comment (Jasperkroes) [inline src/main/java/camera/CameraActivity.java]: ?

Comment (egedebruin) [reply src/main/java/camera/CameraActivity.java]: !

File: src/main/java/gui/controllers/TimerController.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package gui.controllers;
+
+import java.util.concurrent.TimeUnit;
+import javafx.animation.AnimationTimer;
+
+/**
+ * TimerController class for controlling GUI elements.
+ */
+public class TimerController {
+
```
Comment (Jasperkroes) [inline src/main/java/gui/controllers/TimerController.java]: waarom extend dit geen controller

Comment (egedebruin) [reply src/main/java/gui/controllers/TimerController.java]: Omdat het geen controller is

Comment (Jasperkroes) [reply src/main/java/gui/controllers/TimerController.java]: waarom heet het dan controller?

Comment (egedebruin) [reply src/main/java/gui/controllers/TimerController.java]: Omdat ik geen betere naam wist, weet jij een betere naam?

Comment (Jasperkroes) [reply src/main/java/gui/controllers/TimerController.java]: uhm

Comment (Jasperkroes) [reply src/main/java/gui/controllers/TimerController.java]: ff denken hoor

Comment (Jasperkroes) [reply src/main/java/gui/controllers/TimerController.java]: TimerController class for controlling GUI elements.

**Pull request merged**